docs/content/en/docs/cluster-operator/vms-cluster-lifecycle.md

```diff
@@ -129,11 +129,13 @@ If you just modify the IP and apply the manifest directly, you will get a kubern
 
 Here are some optional parameters you may care about and the setting position:
 
-| parameters | setting-position                            |
-| :-----|:--------------------------------------------| 
-| KubeVersion | KubeadmControlPlane.Spec.Version            | 
-| PodCIDR | Cluster.Spec.ClusterNetwork.Pods.CidrBlocks | 
-| CNIPlugin | CustomCluster.CNI.Type                      | 
+| parameters    | setting-position                                                |
+|:--------------|:----------------------------------------------------------------| 
+| KubeVersion   | KubeadmControlPlane.Spec.Version                                | 
+| PodCIDR       | Cluster.Spec.ClusterNetwork.Pods.CidrBlocks                     | 
```

Comment:
can you update link these to api docs?

Reply:
I attempted to run "make gen-api-doc", but no new content was generated automatically .
The variable is not located in customcluster/custommachine, but in the third-party cluster/kcp.
Therefore, may be I need to manually write documentation to describe these variables？